### PR TITLE
RVAT-535_PossibilityViewMainPage

### PR DESCRIPTION
**Pull request by MaxKovalets**
Please check my test RVAT-540_checkExistenceOfTheTitle

### CODE REVIEW

File: src/test/java/rv/atqc/gttesting/possibility_view_main_page/PossibilityViewMainPageChrome.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import io.github.bonigarcia.wdm.ChromeDriverManager;
+
+public class PossibilityViewMainPageChrome{
+
+	public WebDriver driver;
+	
+	@Test
+	public void checkExistenceOfTheTitle(){
+        driver.manage().timeouts().implicitlyWait(1, TimeUnit.SECONDS);
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/possibility_view_main_page/PossibilityViewMainPageChrome.java]: I suggest using the explicit waiter... please find the links in the study_res channel

File: src/test/java/rv/atqc/gttesting/possibility_view_main_page/PossibilityViewMainPageChrome.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+		driver.get("https://green-tourism.herokuapp.com/");
+		WebElement titleElement = driver.findElement(By.className("parallax-title"));
+		Assert.assertTrue(titleElement.isDisplayed());
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/possibility_view_main_page/PossibilityViewMainPageChrome.java]: your method name doesn't match your assertion logic. please provide another title or change assert condition.

**Review by dmatsev — COMMENTED** [review]
please make changes and notify me about it

File: src/test/java/rv/atqc/gttesting/possibility_view_main_page/PossibilityViewMainPageChrome.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+	public WebDriver driver;
+	public WebDriverWait wait;
+	
+	@BeforeClass
+	public void before(){		
+		ChromeDriverManager.getInstance().setup();
+		driver=new ChromeDriver();
+		driver.get("https://green-tourism.herokuapp.com/");
+		wait = new WebDriverWait(driver,10);
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/possibility_view_main_page/PossibilityViewMainPageChrome.java]: this magic number. please move it to the MAX_WAIT_TIME const.

**Review by dmatsev — APPROVED** [review]

**Comment by TyshchenkoRoma** [comment]
good

**Review by TyshchenkoRoma — APPROVED** [review]

**Review by MelnykNikita — APPROVED** [review]

**Pull request closed without merging**